include/torch-mlir/Dialect/Torch/Utils/TorchUpstream.h

```diff
@@ -70,6 +70,13 @@ struct ResultTypeState {
 ScalarType result_type(const ResultTypeState &in_state);
 ScalarType promote_skip_undefined(ScalarType a, ScalarType b);
 
+// These constants control the reduction behavior of the loss functions.
```

Comment:
nit: can you change the comment to similar format as the other comments?

Reply:
Done, thanks.